### PR TITLE
Webview feature

### PR DESCRIPTION
**Pull request by jordanwade**
## DESCRIPTION
Set up a webview feature that will allow the embedding of Spotify playlists or other url embeds.

### What does this PR do, or why is it needed?

Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-05 at 15 33 02

### How do I test this PR?

Check out the story `ui-connected/WebviewFeature example`

### What automated tests did you write?

Added webview feature tests to `ContentSingleFeaturesConnected`

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by conrad-vanl** [comment]
As much as I'd love to review this, this is totally within @vinnyjth's wheelhouse and would like to kick this over to him. I'm more the give you balloons and cake person with an `A+` sticker as from my knowledge of features, this looks overall great!

File: packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/WebviewFeature.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import PropTypes from 'prop-types';
+import { SideBySideView, H3, UIText, styled } from '@apollosproject/ui-kit';
+import { WebView } from 'react-native-webview';
+import { Platform, View, TouchableOpacity, Linking } from 'react-native';
+// import { SafeAreaView } from 'react-navigation';
+
+const StyledCard = styled(
+  ({ theme }) => ({
+    backgroundColor: theme.colors.black,
+    height: 400,
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/WebviewFeature.js:11]: Curious about this height. I imagine it's really hard to decide on what this value should be, and 400 is totally right for the Spotify View. I think this is fine for now, because we have an override for this component (good catch on that, btw) but in the future maybe we should expose height as a prop?

Comment (conrad-vanl) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/WebviewFeature.js:11]: We could easily expose `height` as a prop now and just set the `defaultProp` for it to 400.

Comment (jordanwade) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/WebviewFeature.js:11]: Cool, I didn't like putting a fixed height. I'll update to just expose the height and default it to 400. Good idea!

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/WebviewFeature.js:11]: Agh! I merged this too soon. Feel free to do this in a follow up PR

Comment (jordanwade) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/WebviewFeature.js:11]: Actually, since the height is set in the styles and the styles for the webview feature card are exposed with `ui-connected.WebviewFeature.StyledCard` should we still expose the height in the props too?

image

Comment (conrad-vanl) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/WebviewFeature.js:11]: overriding in styles would allow a church to override it globally, instead of on a case-per-case basis. Technically a church could still use a style override in a case-per-case basis since style overrides can be functions, and when they are a function, they receive the same props of the component. Generally we'd opt for both in case like this. 

I think for now we can just leave it as is and add the prop when someone requests it.

Comment (jordanwade) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/WebviewFeature.js:11]: Again good point, thank ya! Adding it 👍
